web/pipelines/steps-web-run-a11y-tests.yaml

```diff
@@ -8,3 +8,12 @@ steps:
     inputs:
       archiveFilePatterns: '${{ parameters.workspaceDir }}/a11y-tests/a11y-tests.zip'
       destinationFolder: '${{ parameters.workspaceDir }}/a11y-tests-files'
+  - task: qetza.replacetokens.replacetokens-task.replacetokens@3
```

Comment:
Please use latest version of this task `qetza.replacetokens.replacetokens-task.replacetokens@6`.

Examples can be found in `steps-run-terraform.yaml`

Reply:
Please see the examples - the inputs are different between v3 and v6

Reply:
ahh yes. i have updated it now.